ingestion_tools/scripts/importers/annotation.py

```diff
@@ -261,19 +274,29 @@ def get_metadata(self, output_prefix: str) -> list[dict[str, Any]]:
 class SegmentationMaskAnnotation(VolumeAnnotationSource):
     shape = "SegmentationMask"  # Don't expose SemanticSegmentationMask to the public portal.
     mask_label: int
+    rescale: bool = False
+    threshold: float | None
```

Comment:
just a TODO for later that we might want to see if we can/should de-duplicate some of the identical code in SemanticSegmentationMask vs SegmentationMask

Reply:
Agreed, I'm not sure why there are two classes to be completely honest. They seem to do the same thing.

Reply:
Added a note.